openfisca_web_api_preview/app.py

```diff
@@ -109,8 +109,7 @@ def calculate():
 
             variable = tax_benefit_system.get_variable(variable_name)
             if variable.value_type == Enum:
-                entity_result = variable.possible_values._vars[entity_result]
-
+                entity_result = entity_result.name
```

Comment:
Type conversion in the same variable might lead to problems later. 
What about setting the condition on the `Enum` before `entity_result` init?

Reply:
Better ?